Commons/core/src/main/i18n/templates/pgm/PGMUI.properties

```diff
@@ -352,3 +352,5 @@ stats.ui.cores = Cores Leaked:
 stats.ui.monuments = Monuments Destroyed: 
 stats.ui.teamkills = TK: 
 
+autojoin.starting = Match is starting in {0} seconds! Left click the hat to cancel autojoin!
+autojoin.cancelled = You have cancelled autojoin!
```

Comment:
`You will now observe the match. Right click the helmet if you want to join again!`